<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanFdForm</class>
 <widget class="QWidget" name="CanFdForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="devLabel">
       <property name="text">
        <string>Device</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="devHLayout">
       <item>
        <widget class="QLineEdit" name="devLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="devPushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="clkFreqLabel">
       <property name="text">
        <string>Clock Frequency(MHz)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="clkFreqLineEdit"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="arbitLabel">
           <property name="text">
            <string>Arbitration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="arbitBaudLabel">
           <property name="text">
            <string>Baudrate Bits/second</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="arbitBaudLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="arbitPreLabel">
           <property name="text">
            <string>Prescaler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="arbitPreLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="arbitTseg1Label">
           <property name="text">
            <string>Tseg1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="arbitTseg1LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="arbitTseg2Label">
           <property name="text">
            <string>Tseg2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="arbitTseg2LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="arbitSjwLabel">
           <property name="text">
            <string>Sjw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="arbitSjwLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="dataLabel">
           <property name="text">
            <string>Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dataBaudLabel">
           <property name="text">
            <string>Baudrate Bits/second</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataBaudLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="dataPreLabel">
           <property name="text">
            <string>Prescaler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataPreLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="dataTseg1Label">
           <property name="text">
            <string>Tseg1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataTseg1LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="dataTseg2Label">
           <property name="text">
            <string>Tseg2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataTseg2LineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="dataSjwLabel">
           <property name="text">
            <string>Sjw</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dataSjwLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>124</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
